MVC/models/articlesModels.js

```diff
@@ -11,53 +11,70 @@ exports.selectArticleById = (article_id) => {
   });
 };
 
-exports.selectAllArticles = () => {
-  const sql = `SELECT articles.article_id, articles.title, articles.topic, 
+exports.selectAllArticles = (topic) => {
+  const arrTopics = ["mitch", "cats", "paper"];
```

Comment:
We don't want to be hard coding these values at the server level, what would happen if new topics were added to the front end in the future? We should allow our logic to be inferred from the database response rather than hard coded values here.